src/test-helpers/e2e-helpers/initPage.ts

```diff
@@ -0,0 +1,32 @@
+import skipTutorialScreen from './skipTutorialScreen'
+import { Browser, Device, Page } from 'puppeteer'
+
+export interface InitPageOptions {
+  puppeteerBrowser?: Browser,
+  url?: string,
+  skipTutorial?: boolean,
+  emulatedDevice?: Device,
+}
+
+/**
+ * Create a new incognito context and page.
+ */
+const initPage = async ({ puppeteerBrowser = browser, url = 'http://localhost:3000', emulatedDevice, skipTutorial = true }: InitPageOptions) => {
+  const context = await puppeteerBrowser.createIncognitoBrowserContext()
+  const page: Page = await context.newPage()
+
+  if (emulatedDevice) {
+    await page.emulate(emulatedDevice)
+  }
+  page.on('dialog', async dialog => {
+    await dialog.accept()
+  })
+
+  await page.goto(url)
+
+  skipTutorial && await skipTutorialScreen(page)
+
+  return { page, context }
```

Comment:
Let's just return `page`. It's a cleaner api and we can get the context by calling `page.browserContext()`.